databricks/koalas/base.py

```diff
@@ -299,10 +301,25 @@ def is_monotonic(self):
 
         >>> ser.rename("a").to_frame().set_index("a").index.is_monotonic
         True
+
+        >>> ser = ks.Series([5, 4, 3, 2, 1], index=[1, 2, 3, 4, 5])
+        >>> ser.is_monotonic
+        False
+
+        >>> ser.index.is_monotonic
+        True
         """
+        from databricks.koalas.frame import DataFrame
+        from databricks.koalas.series import _col
```

Comment:
This imports seem not needed.

Reply:
you're right, i fixed whole logic. Thanks!!